_source/logzio_collections/_log-sources/aws-cost-and-usage-report.md

```diff
@@ -18,6 +18,17 @@ shipping-tags:
   - aws
 order: 830
 ---
+
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Manual configuration](#manual-configuration)
+* [Deployment using a module](#module-deployment)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="manual-configuration">
+
 AWS Cost and Usage Reports function tracks your AWS usage and provides estimated charges associated with your account. This integration allows you to ship logs from your AWS Cost and Usage Reports to your Logz.io account.
```

Comment:
I think this description and the info box should be in a separate "Overview" tab.